agent/engine/docker_image_manager.go

```diff
@@ -282,14 +283,14 @@ func (imageManager *dockerImageManager) removeImageState(imageStateToBeRemoved *
 
 func (imageManager *dockerImageManager) getCandidateImagesForDeletion() []*image.ImageState {
 	if len(imageManager.imageStatesConsideredForDeletion) < 1 {
-		seelog.Debugf("Image Manager: Empty state!")
+		logger.Debug("Image Manager: Empty state!")
 		// no image states present in image manager
 		return nil
 	}
 	var imagesForDeletion []*image.ImageState
 	for _, imageState := range imageManager.imageStatesConsideredForDeletion {
 		if imageManager.isImageOldEnough(imageState) && imageState.HasNoAssociatedContainers() {
-			seelog.Infof("Candidate image for deletion: [%s]", imageState.String())
+			logger.Debug("Candidate image for deletion", imageState.Fields())
```

Comment:
these messages are logged multiple times when the image is actually removed, and then again on each "removal" loop, so making it debug